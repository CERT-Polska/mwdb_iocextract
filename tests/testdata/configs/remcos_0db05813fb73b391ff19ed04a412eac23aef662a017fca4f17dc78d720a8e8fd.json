{"c2": [{"host": "enmark81.duckdns.org:4045", "password": ""}], "type": "remcos", "raw_cfg": {"in-blob": "a301a84866ad5f8e16af6b20388ded79b0af10a3824deae212fc445ebb9a8ecb"}}