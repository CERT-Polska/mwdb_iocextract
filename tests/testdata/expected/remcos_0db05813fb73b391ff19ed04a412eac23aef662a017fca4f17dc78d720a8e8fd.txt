NetLoc enmark81.duckdns.org:4045
